crates/uv-distribution/src/source/mod.rs

```diff
@@ -1556,7 +1556,7 @@ async fn read_pkg_info(
     Ok(metadata)
 }
 
-/// Read the [`Metadata23`] from a source distribution's `pyproject.tom` file, if it defines static
+/// Read the [`Metadata23`] from a source distribution's `pyproject.toml` file, if it defines static
```

Comment:
Could commit this to `main` if you want to reduce the PR.

Reply:
https://github.com/astral-sh/uv/pull/3323